Steps/Education.cs

```diff
@@ -0,0 +1,62 @@
+﻿using NUnit.Framework;
+using TechTalk.SpecFlow;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    public class Education
+    {
+
+        MarsQA_1.SpecflowPages.Pages.EducationProfile EducationPage = new SpecflowPages.Pages.EducationProfile();
+
+        [Given(@"User enter UniversityName as ""(.*)""")]
+        public void GivenSellerEnterUniversityNameAs(string p0)
+        {
+            EducationPage.University = p0;

```

Comment:
Why are you passing the data to properties instead as a parameter into the method?

Reply:
this would be more easier I guess. For example for Language page if I have to pass parameters by method In Step definition I will have to use scenario context class which require a lot of new steps as below.
In step definition
public void WhenUserEnterLanguageAsAndLevelAs(string language, string level)
        {
            ScenarioContext.Current.Add("language", language);
            ScenarioContext.Current.Add("level", level);
            language = ScenarioContext.Current["language"].ToString();
            level= ScenarioContext.Current["level"].ToString();
            LanguagePage.AddLanguage(language, level);
        }

and in language page class

   public void AddLanguage(string Languages, string Levels)
        {
            LanguageTab.Click();
            AddNewButton.Click();
            LanguageName.SendKeys(Languages);
            var selectElement = new SelectElement(SelectLevel);
            selectElement.SelectByText(Levels);
            AddButton.Click();
        }

So I think this would be a more easier approach.